integration-tests/contract-xcm/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
```

Comment:
Since you started this PR, we've split `integration-tests` into an internal and a public folder. Please move `contract-xcm` into the public one.